### PR TITLE
chore(DTFS2-7592): merge release-2.4.0 to main

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction :pencil2:
Merge `release-2.4.0` release branch to `main`.

### CODE REVIEW

File: azure-functions/acbs-function/src/functions/acbs-amend-facility.js [hunk 1]
```diff
@@ -138,23 +138,28 @@ df.app.orchestration('acbs-amend-facility', function* amendFacility(context) {
       amendments,
     });
 
+    /**
+     * TODO: DTFS2-7581
+     * Below have been disabled for release 2.4.0.
```
Comment (ttbarnes) [inline azure-functions/acbs-function/src/functions/acbs-amend-facility.js:143]: Can these be enabled now?

**Review by ttbarnes — CHANGES_REQUESTED** [review]

File: azure-functions/acbs-function/src/functions/acbs-amend-facility.js (hunk 1 above)
Comment (abhi-markan) [reply azure-functions/acbs-function/src/functions/acbs-amend-facility.js:143]: Not until QA ACs have been confirmed.

**Review by ttbarnes — APPROVED** [review]

File: azure-functions/acbs-function/src/functions/acbs-amend-facility.js [hunk 2]
```diff
@@ -138,23 +138,28 @@ df.app.orchestration('acbs-amend-facility', function* amendFacility(context) {
       amendments,
     });
 
+    /**
+     * TODO: DTFS2-7581
```
Comment (Zainzzkk) [inline azure-functions/acbs-function/src/functions/acbs-amend-facility.js:142]: is this meant to come to main?

**Review by ChristianMcc — APPROVED** [review]

File: azure-functions/acbs-function/src/functions/acbs-amend-facility.js (hunk 2 above)
Comment (abhi-markan) [reply azure-functions/acbs-function/src/functions/acbs-amend-facility.js:142]: Yes that's correct.

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**